### PR TITLE
fix: Update install method for Bun

### PR DESCRIPTION
**Pull request by Zaid-maker**
### What?
So, i was trying to create a Next.js project with Bun as i was experimenting with it and i got error creating the project `error: could not determine executable to run for package create` so i added `-` after create and works and created by project successfully.

## Before

image

## After
image

### CODE REVIEW

**Review by ijjk — COMMENTED** [review]
See https://github.com/vercel/next.js/pull/64081 seems this is an issue in bun as either variation of this command is supposed to work according to their docs

**Comment by Zaid-maker** [comment]
> See https://github.com/vercel/next.js/pull/64081 seems this is an issue in bun as either variation of this command is supposed to work according to their docs

I Double check by upgrading my Bun and the issue is still same.

**Comment by Zaid-maker** [comment]
Verified on a Linux Machine as well!

image

**Comment by Zaid-maker** [comment]
I read the docs they say both of these work identically might a bug in Bun idk!! Tested on Windows and Linux tho

**Comment by Zaid-maker** [comment]
I have update to show to use both methods... Sounds good?

**Comment by Zaid-maker** [comment]
@ijjk Is it good now? 😄

File: docs/02-app/02-api-reference/06-create-next-app.mdx [hunk 1]
```diff
@@ -26,7 +26,10 @@ pnpm create next-app
 ```
 
 ```bash filename="Terminal"
+# Both methods works Identically
 bunx create next-app
+or
+bunx create-next-app
```
Comment (ijjk) [inline docs/02-app/02-api-reference/06-create-next-app.mdx:32]:
```suggestion
bunx create-next-app
```

Comment (ijjk) [reply docs/02-app/02-api-reference/06-create-next-app.mdx:32]: Let's leave it as this as from testing against latest bun this appears to work correctly while the other doesn't

File: docs/02-app/02-api-reference/06-create-next-app.mdx [hunk 2]
```diff
@@ -26,7 +26,8 @@ pnpm create next-app
 ```
 
 ```bash filename="Terminal"
-bunx create next-app
+# Both methods works Identically
```
Comment (ijjk) [inline docs/02-app/02-api-reference/06-create-next-app.mdx:29]:
```suggestion
```

**Review by ijjk — APPROVED** [review]

**Comment by ijjk** [comment]
Let's leave it as this as from testing against latest bun this appears to work correctly while the other doesn't, thanks!

**Pull request merged**